### ISSUE TITLE
Get-Help Potentially Incorrect Help Topic for New-ModuleManifest

### ISSUE THREAD

**Issue by RickFlist** [PowerShell/PowerShell#4932]
Steps to reproduce
------------------
It appears the help topic for a parameter on New-ModuleManifest is incorrect. I can't be sure, but it doesn't look like the text goes with the parameter
```powershell
Get-Help New-ModuleManifest -Parameter CompatiblePSEditions
```

Expected behavior
-----------------
N/A
```none

```

Actual behavior
---------------
Get-Help returns text for the parameter CompatiblePSEditions that does not appear to match the parameter name. 
```none
-CompatiblePSEditions <String[]>
    You can use this parameter to restrict the aliases that are exported by the module. It can remove aliases from the list of
    exported aliases, but it cannot add aliases to the list.

    If you omit this parameter, New-ModuleManifest creates an AliasesToExport key with a value of * (all), meaning that all aliases
    that are exported by the module are exported by the manifest.

    Required?                    false
    Position?                    named
    Default value                None
    Accept pipeline input?       False
    Accept wildcard characters?  false
```

Environment data
----------------

```powershell
PS>( ( $PSVersionTable.GetEnumerator() | ForEach-Object { ( '{0,26} : {1,-20}' -f $PSItem.Key,($PSItem.Value -join ', ') ) } ) | Out-String -Width 100 ) | Set-Clipboard
                 PSVersion : 5.1.15063.608       
                 PSEdition : Desktop             
      PSCompatibleVersions : 1.0, 2.0, 3.0, 4.0, 5.0, 5.1.15063.608
              BuildVersion : 10.0.15063.608      
                CLRVersion : 4.0.30319.42000     
         WSManStackVersion : 3.0                 
 PSRemotingProtocolVersion : 2.3                 
      SerializationVersion : 1.1.0.1             

```

**Comment by SteveL-MSFT** [comment]
Looks like this was fixed already https://github.com/PowerShell/PowerShell-Docs/commit/6944160b09b2e45ea074e2443d3b326e6fc8c31e#diff-52086eee0a9f9d3b416db384cf54ef38, although it doesn't appear to have been published yet

**Comment by RickFlist** [comment]
Thanks. Was there some way I could have found that without bothering you?

**Comment by SteveL-MSFT** [comment]
@RickFlist good question!  In this case, all doc content issues should be in https://github.com/powershell/powershell-docs repo (not this one).  I looked to see if this was a known issue and it looks like our doc writer already fixed it (in the repo).  In this case, there wasn't an existing issue in that repo, so I just looked at the actual doc (it's under the `reference` folder) and it was already changed.
